org.gpodder.gpodder-adaptive.json

```
@@ -0,0 +1,52 @@
+{
+  "app-id": "org.gpodder.gpodder-adaptive",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
+  "sdk": "org.gnome.Sdk",
+  "command": "gpodder",
+  "rename-icon": "gpodder-adaptive",
+  "copy-icon": true,
+  "rename-desktop-file": "gpodder.desktop",
+  "finish-args": [
+    "--device=dri",
+    "--socket=x11",
+    "--socket=wayland",
```

```suggestion
    "--socket=fallback-x11",
    "--socket=wayland",
```

it it really supports wayland.